lib/scsi_defs.h

```diff
@@ -108,5 +115,29 @@ namespace scsi_defs {
     kCompleted = 0xF
   };
 
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6 Table 58
+  typedef struct InquiryCommand {
```

Comment:
`typedef`ing structs isn't needed in C++

Reply:
changed to `struct`